src/Editor/Util/AssetConverter.cpp

```diff
@@ -30,6 +29,17 @@ void AssetConverter::Convert(const char * filepath, const char * destination, gl
         Log() << aImporter.GetErrorString() << "\n";
     }
 
+    if (importMaterial) {
+        if (aScene->mMeshes[0]->mMaterialIndex >= 0) {
+            aiMaterial* material = aScene->mMaterials[aScene->mMeshes[0]->mMaterialIndex];
+            
+            LoadMaterial(material, aiTextureType_DIFFUSE, materials.albedo);
+            LoadMaterial(material, aiTextureType_NORMALS, materials.normal);
+            LoadMaterial(material, aiTextureType_SPECULAR, materials.roughness);
+            LoadMaterial(material, aiTextureType_REFLECTION, materials.metallic);
+        }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'materialDestinationFilePath' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)